src/data-layer/variables/isAuth.tsx

```diff
@@ -1,5 +1,18 @@
 import { makeVar } from '@apollo/client';
-import { getClientCookie } from '@camp/infra';
+import { getClientCookie, removeClientCookie } from '@camp/infra';
 
 const clientCookie = getClientCookie();
 export const isAuthVar = makeVar(clientCookie.isAuth);
+
+export const loginLocally = () => {
+  const getCookie = getClientCookie();
+
+  if (!getCookie.isAuth) {
+    isAuthVar(true);
+  }
```

Comment:
Why set auth to true When `isAuth` is false? :thinking:

Reply:
I added it temporary. just for completing the login flow . if i'm not mistaken the problem is about that from backend side there is no cookie.